timescaledb/how-to-guides/toolkit/counter_agg.md

```diff
@@ -0,0 +1,164 @@
+# Counter aggregates
+There are two types of metrics that can be recorded in a database: gauges, and
+counters. Gauges fluctuate up and down, like temperature or speed, while
+counters always increase, like the total number of visitors to a website.
+
```

Comment:
Maybe we can take the speed analogy and use this throughout? use a car as our story? To explain them, let's think about a car: we have gauges which can fluctuate up and down, like speed or the fuel gauge, and we have counters like the odometer which always increases and tracks the total number of miles the car has traveled.